tests/regression/test_validator_exit_bus_happy_path.py

```diff
@@ -0,0 +1,226 @@
+import dataclasses
+
+import pytest
+from brownie import web3
+from brownie.convert.datatypes import HexString
+from eth_abi import encode
+
+from utils.config import contracts
+from utils.test.exit_bus_data import encode_data, LidoValidator
+from utils.test.oracle_report_helpers import (
+    wait_to_next_available_report_time, reach_consensus,
+)
+
+
+@dataclasses.dataclass
+class ProcessingState:
+    current_frame_ref_slot: int
+    processing_deadline_slot: int
+    data_hash: HexString
+    data_submitted: bool
+    data_format: int
+    requests_count: int
+    requests_submitted: int
+
+
+@pytest.fixture()
+def stranger(stranger):
```

Comment:
can use oracle committee member instead (we are not assigning this role for the real setup)

Reply:
yes, it's not used, forgot to remove, thanks!